gems/pending/Gemfile

```diff
@@ -24,6 +25,7 @@ gem "fog",                     "~>2.0.0.pre.0",     :require => false
 gem "fog-core",                "!=1.31.1",          :require => false
 gem "httpclient",              "~>2.5.3",           :require => false
 gem "kubeclient",              "=0.8.0",            :require => false
+gem "hawkular-client",         "=0.1.1",            :require => false
```

Comment:
Prefer `~>` over `=` if you can.  It makes it easier to keep patch releases up to date...particularly if the gem follows SemVer (which most gems do).  Same goes for `kubeclient` for that matter, which, since you own, should be very easy to follow SemVer.